### ISSUE TITLE
Making FLAIR case insensitive

### ISSUE THREAD

**Issue by igormis** [flairNLP/flair#2380]
A clear and concise description of what you want to know.
I trained flair model using these embeddings:
`embedding_types: List[TokenEmbeddings] = [
    WordEmbeddings('glove'),
    FlairEmbeddings('news-forward'),
    FlairEmbeddings('news-backward')
]`
and trained using this code:
`
embeddings: StackedEmbeddings = StackedEmbeddings(embeddings=embedding_types)
from flair.models import SequenceTagger

tagger: SequenceTagger = SequenceTagger(hidden_size=256,
                                        embeddings=embeddings,
                                        tag_dictionary=tag_dictionary,
                                        tag_type=tag_type,
                                        use_crf=True)

from flair.trainers import ModelTrainer

trainer: ModelTrainer = ModelTrainer(tagger, corpus)

trainer.train('resources/taggers/example-ner',
              train_with_dev=True,
              patience=2,
              max_epochs=250)
`

However I have problems with case sensitivity concerning the output labels.
For instance if I run this example:
**Facebook, Twitter, LinkedIn secure Afghan users' accounts amid takeover**
the output is good:
`
{
  "intent": {
    "name": "",
    "confidence": 0
  },
  "entities": [
    {
      "start": 0,
      "end": 8,
      "value": "Facebook",
      "entity": "company"
    },
    {
      "start": 10,
      "end": 17,
      "value": "Twitter",
      "entity": "company"
    },
    {
      "start": 19,
      "end": 27,
      "value": "LinkedIn",
      "entity": "company"
    },
    {
      "start": 35,
      "end": 41,
      "value": "Afghan",
      "entity": "location"
    },
    {
      "start": 42,
      "end": 48,
      "value": "users",
      "entity": "financial_topic"
    },
    {
      "start": 63,
      "end": 71,
      "value": "takeover",
      "entity": "event"
    }
  ],
  "text": "Facebook, Twitter, LinkedIn secure Afghan users' accounts amid takeover"
}
`
However If I change the input (only add capital letters to each word:
**Input: Facebook, Twitter, LinkedIn Secure Afghan Users' Accounts Amid Takeover**
The output is far from good:
`{
  "intent": {
    "name": "",
    "confidence": 0
  },
  "entities": [
    {
      "start": 0,
      "end": 8,
      "value": "Facebook",
      "entity": "company"
    },
    {
      "start": 10,
      "end": 17,
      "value": "Twitter",
      "entity": "company"
    },
    {
      "start": 19,
      "end": 27,
      "value": "LinkedIn",
      "entity": "company"
    },
    {
      "start": 28,
      "end": 41,
      "value": "Secure Afghan",
      "entity": "company"
    },
    {
      "start": 42,
      "end": 48,
      "value": "Users",
      "entity": "company"
    }
  ],
  "text": "Facebook, Twitter, LinkedIn Secure Afghan Users' Accounts Amid Takeover"
}`
How do I change this behavior?

**Comment by helpmefindaname** [comment]
you could use `embedding_types: List[TokenEmbeddings] = [ WordEmbeddings('glove'), FlairEmbeddings('news-forward', is_lower=True), FlairEmbeddings('news-backward', is_lower=True) ]`
to make the FLairEmbeddings case insensitive.

However as they are trained case sensitive as the casing is usually a good feature, you might want to finetune the embeddings on lowercased texts
see: https://github.com/flairNLP/flair/blob/master/resources/docs/TUTORIAL_9_TRAINING_LM_EMBEDDINGS.md
